### DISCUSSION TITLE
Docu section extras sync online

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#229] [Q&A] [answered]
Is this still valid ?
QRcode download works fine for me .
I checked the service " boothupload.service"  is not running on my PI

**Comment by mgineer85** [comment]
QR code works as is, no additional service needed.
Only if you want more advanced solutions.

**Reply by Striker62** [reply]
did i install photobooth-app not correct. I dio not have a boothupload.sh in my installation folder ?
My default folder is photobooth but that should not be an issue ?

**Reply by mgineer85** [reply, marked as answer]
I suggest not use boothupload. It’s advanced use for experienced users.

**Reply by Striker62** [reply]
ok thanks
